packages/apps/public/locales/it/translation.json

```
@@ -997,50 +1281,84 @@
+  "unbonding": "svincolando",
+  "unchecked weight for this call": "peso non verificato per questo richiamo",
+  "unstake threshold": "soglia di stake",
+  "until block": "fino al blocco",
   "update on #{{index}}": "aggiornamento su #{{index}}",
   "upgradable extensions": "estensioni aggiornabili",
+  "use on any network": "usa su qualsiasi network",
+  "using my account": "usando il mio account",
   "using the following data": "usando i dati seguenti",
   "using the selected account": "usando l'account selezionato",
   "validation": "validazione",
+  "validator to query": "validator da interrogare",
   "validators": "validatori",
+  "validators/nominators": "validators/nominators",
   "value": "valore",
+  "value bonded": "valore vincolato",
   "verify using address": "verifica usando l'indirizzo",
   "version {{version}}": "versione {{version}}",
+  "vested": "maturato",
+  "vesting": "maturazione",
   "via Council": "tramite Consiglio",
   "via Council/Motions": "tramite Consiglio/Mozioni",
+  "via Council/Vote": "via Voto/Consiglio",
   "via Democracy": "tramite Democrazia",
   "via Democracy/Dispatch": "tramite Democrazia",
+  "via Democracy/Vote": "via Voto/Democrazia",
   "via Society": "tramite Società",
   "via Staking": "tramite Staking",
+  "via Staking/Bond": "via Staking/Vincolo",
   "via Staking/Slashed": "tramite Staking/Slashes",
   "via Treasury": "tramite Tesoreria",
+  "via Vesting": "via Maturazione",
   "vote for candidate": "vota per il candidato",
   "vote for defender": "vota per il difensore",
   "vote from account": "vota dall'account",
+  "vote value": "valore del voto",
+  "vote with account": "vota con l'account",
   "votes": "voti",
   "voting account": "account votante",
+  "voting balance": "saldo del voto",
   "voting end": "fine del referendum/voto",
   "voting period": "finestra di voto",
   "voting round": "sessione di voto",
+  "waiting": "in attesa",
   "web": "web",
   "website": "website",
   "with an index of": "con un indice di",
+  "with unchecked weight": "con peso non verificato",
+  "with weight override": "con peso sovrascritto",
+  "with/without identity": "con/senza identità",
+  "without unchecked weight": "senza peso verificato",
   "yes": "si",
   "your current password": "la tua password attuale",
   "your new password": "la tua nuova password",
   "{{blocks}} blocks": "{{blocks}} blocchi",
   "{{count}} key/value pairs encoded for submission": "{{count}} coppie chiave/valore codificate per l'invio",
+  "{{count}} own stashes": "{{count}} stashes di proprietà",
+  "{{currency}} average": "{{currency}} media",
+  "{{currency}} clipped": "{{currency}} approssimata",
+  "{{currency}} rewards": "{{currency}} ricompense",
+  "{{currency}} slashed": "{{currency}} slashed",
+  "{{currency}} total": "{{currency}} totale",
   "{{days}} days": "{{days}} giorni",
   "{{done}}/{{total}}, {{progress}}% done": "{{done}}/{{total}}, {{progress}}% done",
+  "{{d}} days": "{{d}} giorni",
```

```suggestion
  "{{days}} days": "{{days}} giorni",
```

Small updates here and below, the keys were changed today to actually work as expected.